src/main/java/org/neocoder/AuthService.java

```diff
@@ -0,0 +1,14 @@
+package org.neocoder;
```

Comment:
The package should be "org.neocoder.authservice": src/main/org/neocoder/authservice/ ..